viewer/src/repository/cad/CachedRepository.ts

```diff
@@ -77,7 +83,21 @@ export class CachedRepository implements Repository {
         );
         return merge(
           cachedSectorObservable.pipe(flatMap(wantedSector => this._modelDataCache.get(this.cacheKey(wantedSector)))),
-          uncachedSectorObservable.pipe(this.loadSectorFromNetwork()),
+          uncachedSectorObservable.pipe(
+            flatMap(
+              async _ => this._semaphore.acquire(),
```

Comment:
Comment on the logic of whats going on here (e.g. that acquire() returns a value that we filter on further down)